tests/filesystem/test_filesystem.py

```diff
@@ -174,16 +177,23 @@ def _copy(item: FileItemDict):
             parquet_reader.with_name("parquet_example"),
             downloader.with_name("listing"),
             csv_reader.with_name("csv_example"),
+            csv_duckdb_reader.with_name("csv_duckdb_example"),
         ]
     )
     assert_load_info(load_info)
     assert load_table_counts(
-        pipeline, "jsonl_example", "parquet_example", "listing", "csv_example"
+        pipeline,
+        "jsonl_example",
+        "parquet_example",
+        "listing",
+        "csv_example",
+        "csv_duckdb_example",
     ) == {
         "jsonl_example": 1034,
         "parquet_example": 1034,
         "listing": 10,
         "csv_example": 1270,
+        "csv_duckdb_example": 1272,
```

Comment:
why do we have two more records? maybe we read headers as data or vice versa? please check it out

Reply:
It's because a header in one of the files includes commas - it's not parsed correctly. By the very basic CSV standard, such lines must be escaped with double quotes (otherwise, many parsers will not deal with it). Added them:

![image](https://github.com/dlt-hub/verified-sources/assets/26030071/cf0f75ee-4910-42b3-850b-de1147d5bb6e)